SwiftCompilerSources/Sources/Optimizer/FunctionPasses/AsyncDemotion.swift

```diff
@@ -590,7 +590,7 @@ private func analyzeForDemotion(_ function: Function) -> Node? {
   // because it may be a caller of a function that can be demoted.
 
   var executors = Set<Executor>()
-  var knownAsyncApplys: [ApplySite] = []
+  var knownAsyncApplies: [ApplySite] = []
```

Comment:
I would call this `knownAsyncApplySites` instead.

Reply:
Fixed!